manager/integration/deploy/test.yaml

```diff
@@ -43,12 +43,13 @@ spec:
   containers:
   - name: longhorn-test-pod
     image: longhornio/longhorn-manager-test:07dcd20
-#    args: [
+    args: [
+      "--skip-base-image-test"
 #           "-x", "-s",
 #           "-m", "coretest",
 #           "-k", "test_recurring_job",
 #           "--skip-recurring-job-test",
-#           ]
+    ]
```

Comment:
`"--include-base-image-test"`, make it `False` by default in the conftest.py.
Also, put it after `--skip-recurring-job-test`.

Then we can avoid unnecessary change to the line of `args`

Reply:
Isn't the flag already `False` by default based on how `Python` implements `argparse`?

Reply:
IIUC, if the flag is omitted, it will use the default value defined in the `conftest.py`. Even if it's `False` by default (in the `conftest.py`), it's better to spell it out explicitly rather than let the one who read the code guessing.

Reply:
So the way `argparse` works for handling the `default` value in this situation is this: if `default` isn't specified and `action` is set to either `store_true` or `store_false`, `argparse` will use the opposite as the `default`. So if the option is set to `store_true`, the `default` will be `False` and vice versa.

It probably does make sense to specify the `default` explicitly though for anyone reading the code so they don't have to go digging up the documentation for `argparse`.